lib/api/units/validateStatusCode.js

```diff
@@ -0,0 +1,30 @@
+const { TextDiff } = require('../../validators/text-diff');
+
+const APIARY_STATUS_CODE_TYPE = 'text/vnd.apiary.status-code';
+
+function normalizeStatusCode(statusCode) {
+  return String(statusCode).trim();
```

Comment:
Nitpick: I'm thinking, if the status code is undefined or null, it might be nicer to coerce it to empty string instead of `'null'` or `'undefined'`, so it looks better in the diff/error message. What about something like this?

```js
return statusCode == null ? '' : new String(statusCode).trim();
```

Do you think it's worth?

Reply:
Definitely. Added this change and unit tests to cover `normalizeStatusCode`.